src/java.base/share/classes/jdk/internal/util/OperatingSystem.java

```diff
@@ -84,7 +85,7 @@ public enum OperatingSystem {
     ;
 
     // The current OperatingSystem
-    private static final OperatingSystem CURRENT_OS = initOS();
+    private static @Stable OperatingSystem CURRENT_OS;
```

Comment:
Could this be a `StableValue.supplier()`?

Reply:
I would rather optimize MethodHandleAccessorFactory than defining a new class for this...

Reply:
Rolled back per @jaikiran.